config/kafka-monitor.properties

```diff
@@ -77,62 +80,15 @@
   },
 
 
-  "reporter-service": {
-    "class.name": "com.linkedin.kmf.services.DefaultMetricsReporterService",
+  "signalfx-service": {
```

Comment:
If we intend to contribute this upstream, I don't think you can mess around with the default configuration this much. Maybe provide an example of how to configure the `signalfx-service` in the class Javadoc of `SignalFxMetricsReporterService`?

Reply:
yes make sense